### ISSUE TITLE
Update documentation to include listing on pypi as "powerline-status".

### ISSUE THREAD

**Issue by rdlester** [powerline/powerline#1251]
Powerline is currently installable through pip via the name "powerline-status", but the docs still state that this is impossible. Could the installation instructions be updated to reflect this?

**Comment by ZyX-I** [comment]
You should show where this is stated.

**Comment by rdlester** [comment]
Seems like most of the installation section in the ReadTheDocs site is out of date. The now-false statement is made in https://powerline.readthedocs.org/en/latest/installation.html#pip-installation

**Comment by rdlester** [comment]
Most sections of the installation guide seem slightly confusing or out of date.

https://powerline.readthedocs.org/en/latest/installation.html#pip-installation mentions the pypi release but only at the very end of the section. Worse, the opening sentence of the section is "This project is currently unavailable from PyPI due to a naming conflict with an unrelated project." http://powerline.readthedocs.org/en/master/installation/osx.html should be similarly updated.

http://powerline.readthedocs.org/en/master/installation/linux.html mentions the pypi listing but after the install-from-head pip command and in a footnote.

It seems like the pypi release installation method (`pip install powerline-status`) should be listed before the pip commands that will install a dev version from head. Installation will always be a little involved, but installing from pypi will at least make updating easier for users not actively developing the project.

Also, the git URL is still listed as Lokaltog/powerline. Looks like that redirects to powerline/powerline and still works fine, but it wouldn't hurt to update.

If you agree with these changes, I can open a pull request updating the docs. Hopefully that will update powerline.readthedocs.org?

**Comment by ZyX-I** [comment]
Thanks. /master/ URLs cannot be updated until next release and that would be 2.0. /latest/ URLs should not mention Lokaltog URLs because they are based on documentation from develop branch.

**Comment by rdlester** [comment]
Whoops, didn't mean for there to be a difference between "master" and "latest".
Your commit fixes the linux page, but the main page and osx pages still need updating. I'll open a pr based on your fix.

**Comment by ZyX-I** [comment]
? I have edited all of the installation pages, including main and OS X. Just I did not do this in *one* commit. 08.01.2015, 01:39, "Ryan Lester" <notifications@github.com>:Whoops, didn't mean for there to be a difference between "master" and "latest". Your commit fixes the linux page, but the main page and osx pages still need updating. I'll open a pr based on your fix.—Reply to this email directly or view it on GitHub.
